### PR TITLE
add vs user-access command

### PR DESCRIPTION
**Pull request by caberos**
[#ussue1739](https://github.com/softlayer/softlayer-python/issues/1739)

image

### CODE REVIEW

File: SoftLayer/CLI/virt/access.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+"""Command lines which modify power states."""
```
Comment (allmightyspiff) [inline SoftLayer/CLI/virt/access.py:1]:
```suggestion
"""Get user access details a virtual server.""
```

File: SoftLayer/fixtures/SoftLayer_Virtual_Guest.py [hunk 2]
```diff
@@ -920,3 +920,23 @@
...
+
+getBrowserConsoleAccessLogs = [
+    {
+        "createDate": "2022-03-07T05:32:23-06:00",
+        "eventType": "CONNECTED",
+        "id": 509636,
+        "message": "User connected",
+        "sourceIp": "44.200.9.0",
+        "sourcePort": 47097,
+        "username": "307608_sebastian.chaparro@ibm.com"
```
Comment (allmightyspiff) [inline SoftLayer/fixtures/SoftLayer_Virtual_Guest.py:932]: Dont put real emails/usernames in the fixtures, compliance doens't like that.
```suggestion
        "username": "testuser@ibm.com"
```

File: SoftLayer/fixtures/SoftLayer_Virtual_Guest.py [hunk 3]
```diff
@@ -920,3 +920,23 @@
...
+    },
+    {
+        "createDate": "2022-03-07T07:58:27-06:00",
+        "eventType": "CONNECTED",
+        "id": 509024,
+        "message": "User connected",
+        "sourceIp": "44.200.9.0",
+        "sourcePort": 41185,
+        "username": "307608_sebastian.chaparro@ibm.com"
```
Comment (allmightyspiff) [inline SoftLayer/fixtures/SoftLayer_Virtual_Guest.py:941]:
```suggestion
        "username": "testuser@ibm.com"
```

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Looks good, just change the username in the fixtrues, and the first line comment of the new command.

**Comment by edsonarios** [comment]
Screenshots of the test result
image

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**